script.js

```diff
@@ -19,10 +28,17 @@ const addItemToList = () => {
   listItem.appendChild(deleteItem);
 
   inputField.value = "";
+  storeData();
 };
 
 listContainer.addEventListener("click", (e) => {
   if(e.target.tagName.toLowerCase() === deleteItemTag) {
     e.target.parentElement.remove();
+    storeData()
+  } else if (e.target.tagName.toLowerCase() === inputItemTag) {
+    e.target.toggleAttribute('checked');
+    storeData();
```

Comment:
[minor] `storeData()` could be put to the bottom of the function body to reduce duplications

⚠️Instead of doing this
```
if (condition) {
  // do some and then 
 myFunc(); 
} else {
  // do another and then
  myFunc();
}
```

✅ you could do this:
```
if (condition) {
  // do some
} else {
  // do another
}

myFunc();
```